crates/bevy_reflect/bevy_reflect_derive/src/utility.rs

```diff
@@ -125,7 +125,6 @@ where
 
     member_iter.fold(0, |next_idx, member| match member {
         ReflectIgnoreBehavior::IgnoreAlways => {
-            bitset.insert(next_idx);
```

Comment:
Removed since `ignore`'d fields don't exist as far as reflection is concerned. When it was included, attempting to skip serialization of `ignore`'d fields would result in both the `ignore`'d field and the next one being skipped.